### ISSUE TITLE
Add a Backup/Restore button

### ISSUE THREAD

**Issue by Li-An** [lfk/downloads-router-chrome-extension#3]
Hello, it’s a great extension. I needed something like this for images and it’s handsome it manages domains, extensions and so on.

But it needs a settings backup button so we don’t lose all settings. It seems you don’t have any time to develop this but I had to ask.

Thanks again. It’s the first Chrome’s extension I review.

**Comment by Maverynthia** [comment]
Yes please! Upgraded from 0.8.1 and all my stuff is gone. have to manually port it over T_ T
